packages/back-end/src/scim/users/deleteUser.ts

```diff
@@ -0,0 +1,34 @@
+import { Response } from "express";
```

Comment:
This endpoint isn't used by Okta, only Microsoft Entra, so we may want to remove if the goal of the first iteration is just to support Okta.